vxquery-xtest/src/test/resources/ExpectedTestResults/Simple/US000000001.xml

```diff
@@ -0,0 +1,17 @@
+<!--
```

Comment:
Yes, but the name will not be consistent. Currently, I use fn_doc.xml everywhere.